### PR TITLE
Abacus abmath

### PR DESCRIPTION
**Pull request by siklosid**
@treff7es @sreekanthvga 
computeSampleSize now works.
TODO: more thorough full test.

### CODE REVIEW

**Comment by siklosid** [comment]
@gszjulcsi

File: frontend/app/scripts/services/abmath.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        var sd1 = Math.sqrt(2 * conversion_rate * (1.0 - conversion_rate));
+        var sd2 = Math.sqrt(conversion_rate * (1.0 - conversion_rate) +
+        (conversion_rate + min_effect) * (1.0 - conversion_rate - min_effect));
+
+        return (t_alpha2 * sd1 + t_beta * sd2) * (t_alpha2 * sd1 + t_beta * sd2) / (min_effect * min_effect)
+      };
+
+    this.qNorm = function (p) {
+      debugger;
+      var a0 = 2.50662823884;
```
Comment (gszjulcsi) [inline frontend/app/scripts/services/abmath.js]: :D :D :D
such magic numbers

Comment (gszjulcsi) [inline frontend/app/scripts/services/abmath.js]: And such descriptive variable names!!!

File: frontend/app/scripts/services/abmath.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+        if (p < 0.5) {
+          value = -value;
+        }
+      }
+      /*
+       P <= 0.0 or 1.0 <= P
+       */
+      else {
+        value = NaN;
```
Comment (gszjulcsi) [inline frontend/app/scripts/services/abmath.js]: when do you return NaN?
(won't it coause some troubles later?)

File: frontend/frontend.iml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (gszjulcsi) [inline frontend/frontend.iml]: isn't iml something that only intelliJ needs? @treff7es

File: frontend/app/scripts/services/abmath.js (hunk 1 above)
Comment (siklosid) [inline frontend/app/scripts/services/abmath.js]: indeed :)

File: frontend/app/scripts/services/abmath.js (hunk 2 above)
Comment (siklosid) [inline frontend/app/scripts/services/abmath.js]: Yes, I need to do some error handling here. I just wanted to commit it as soon as possible, so Tamas can start to use it :)

**Pull request merged**